terraform/tfschema/types_terraform.go

```diff
@@ -1250,7 +1348,7 @@ func GenSchemaMetaProvisionTokenV2() map[string]*accessors.SchemaMeta {
 				},
 
 				"allow": {
-					Name:       "Allow",
+					Name:       "allow",
```

Comment:
Due to the inconsistency in `types.proto` this should be capitalized back manually.